gradle.properties

```diff
@@ -20,7 +20,7 @@
 # SOFTWARE.
 #
 
-version=0.11.0-SNAPSHOT
+version=0.11.1-SNAPSHOT
```

Comment:
is this update on purpose? Shouldn't it be splitted into a distinct PR?

Reply:
Nope it was for a local test 😰 
Fixed in [fe8c41c](https://github.com/adevinta/spark-android/pull/1236/commits/fe8c41c37836b6173f6030041f5959f7bfb6ab08)